lib/cache.js

```diff
@@ -13,11 +13,10 @@ util.inherits(Cache, Map);
 
 const dataSize = data => (data && data.length ? data.length : 0);
 
-Cache.prototype.add = function(
-  // Add key-value pair to cache
-  key, // string, key
-  val // value, vaule
-) {
+// Add key-value pair to cache
+//   key - string, key
+//   val - value, vaule
```

Comment:
Typo in 'value'.
Also, type should be 'any', but I'm not sure, maybe we have specific name for it?